flink-core/src/main/java/org/apache/flink/api/common/serialization/DeserializationSchema.java

```diff
@@ -112,5 +114,13 @@ default void deserialize(byte[] message, Collector<T> out) throws IOException {
 		 * @see MetricGroup
 		 */
 		MetricGroup getMetricGroup();
+
+		/**
+		 * Gets the ClassLoader to load classes that are not in system's classpath, but are part of
+		 * the jar file of a user job.
+		 *
+		 * @see UserCodeClassLoader
```

Comment:
```suggestion
		 * Gets the {@link ClassLoader} to load classes that are not in system's classpath, but are part of
		 * the jar file of a user job.
		 *
		 * @see {@link UserCodeClassLoader}
```

Reply:
The entry in `@see` is a link automatically. We do not need to wrap it with `{@link}`.

Reply:
Ok, I learned sth new. 👌